src/mesa/Android.libmesa_dricore.mk

```diff
@@ -50,7 +50,8 @@ LOCAL_WHOLE_STATIC_LIBRARIES := \
 	libmesa_sse41
 LOCAL_CFLAGS := \
 	-msse4.1 \
-       -DUSE_SSE41
+       -DUSE_SSE41 \
+       -Wno-tautological-constant-out-of-range-compare
```

Comment:
I don't think we should hide this warning, we should rather fix the code. One way of doing this would be to introduced UINT_TO_FIXED or similar which would compare against USHRT_MIN.